index.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 
-var Raven = require('raven')
+const raven = require('raven')
 
-exports.register = function (server, options, next) {
-  Raven.config(options.dsn, options.client)
-  server.expose('client', Raven)
-  server.on('request-error', function (request, err) {
-    var baseUrl = request.info.uri ||
-      request.info.host && `${server.info.protocol}://${request.info.host}` ||
+exports.register = function (server, options) {
+  raven.config(options.dsn, options.client)
+  server.expose('client', raven)
+  server.events.on({name: 'request', channels: 'error'}, function (request, event) {
+    const baseUrl = request.info.uri ||
+      (request.info.host && `${server.info.protocol}://${request.info.host}`) ||
```

Comment:
Oh also I'm not sure this works anymore. Looks like the API changed, although the tests are passing. I think this tests needs to be expanded a bit and made into a `t.equals` now that there's no conditional depending on the hapi version:

https://github.com/guischdi/hapi-raven/blob/730acbc24350162499c5821d0169c792a36352a5/test.js#L41

Reply:
Ok, did the change to blue-tape and the `t.equals`...